src/run_tissue_enrichment.py

```diff
@@ -0,0 +1,58 @@
+"""Calculate tissue enrichment."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+import hydra
+
+from etl.common.ETLSession import ETLSession
+from etl.tissue_enrichment.tissue_enrichment import cheers, load_peaks
+
+if TYPE_CHECKING:
+    from omegaconf import DictConfig
+
+
+@hydra.main(version_base=None, config_path=".", config_name="config")
+def main(cfg: DictConfig) -> None:
+    """Run tissue enrichment analysis using the CHEERS method."""
+    etl = ETLSession(cfg)
+    etl.logger.info("Loading data for tissue enrichment calculations...")
+
+    # Loading data
+
+    summary_stats_credible_set = etl.spark.read.format("parquet").load(
+        cfg.etl.tissue_enrichment.inputs.summary_stats_credible_set
+    )
+
+    pics_credible_set = etl.spark.read.format("parquet").load(
+        cfg.etl.tissue_enrichment.inputs.pics_credible_set
+    )
+    tissue_annotations = load_peaks(
+        etl.spark, cfg.etl.tissue_enrichment.inputs.tissue_annotations
+    )
+
+    summary_stats_enrichment = cheers(
+        tissue_annotations,
+        summary_stats_credible_set,
+    )
+
+    pics_enrichment = cheers(
+        tissue_annotations,
+        pics_credible_set,
+    )
```

Comment:
Is there a reason to keep the sumstats and pics data streams separated? Unless there is a fundamental difference, I would suggest to pool the pics credset and sumstats credset datasets into one and run cheers once. Not sure if this move would pose any risk though.